packages/experiments/src/components/Form/inputs/dropdown/FormDropdown.tsx

```diff
@@ -10,20 +10,48 @@ import { IFormContext } from '../../Form';
  * Dropdown input for Form
  */
 export class FormDropdown extends FormBaseInput<
-  number | string,
+  number | string | number[] | string[],
   IFormDropdownProps,
-  IFormBaseInputState<number | string>
+  IFormBaseInputState<number | string | number[] | string[]>
 > {
   constructor(props: IFormDropdownProps, context: IFormContext) {
     super(props, context);
+
+    const { dropdownProps } = props;
+
+    const propsValue = this.props.value !== null && this.props.value !== undefined ? this.props.value : undefined;
+
+    let currentValue: number | string | number[] | string[] | undefined = undefined;
+
+    if (dropdownProps !== undefined && dropdownProps.multiSelect === true) {
+      // If multiSelect is set to true the currentValue should be an array.
+      if (Array.isArray(propsValue)) {
+        currentValue = propsValue;
+      } else if (propsValue !== undefined) {
+        if (typeof propsValue === 'string') {
+          currentValue = new Array<string>(propsValue);
```

Comment:
don't use the array constructor

just assign `[propsValue]`

And you can get rid of the `typeof` then